docs/public/deploying-java-app-with-database.adoc

```diff
@@ -0,0 +1,432 @@
+----
++
+You will see a fully formed URL that can be used in a web browser:
++
+[source,sh]
+----
+[root@pappuses1 jpa]# odo url list
+Found the following URLs for component mysboproj
+NAME               STATE      URL                                           PORT     SECURE     KIND
+mysboproj-9080     Pushed     http://mysboproj-9080.192.168.49.2.nip.io     9080     false      ingress
+[root@pappuses1 jpa]# 
+----
+
+. Use the URL to navigate to the `CreatePerson.xhtml` data entry page and enter requested data:
+URL/CreatePerson.xhtml' and enter a user's name and age data using the the form.
+
+. Click on the *Save* button when complete
+//![Create Person xhtml page](./assets/createPerson.jpg)
+
+Note that the entry of any data does not result in the data being displayed when you click on the "View Persons Record List" link.
+
+==== Binding the database and the application
+
+Now, the only thing that remains is to connect the DB and the application. We will use odo to create a link to the Dev4Devs PostgreSQL Database Operator in order to access the database connection information.
+
+. Display the services available to odo: - You will see an entry for the PostgreSQL Database Operator displayed:
++
+[source,sh]
+----
+$ odo catalog list services
+Operators available in the cluster
+NAME                                             CRDs
+postgresql-operator.v0.1.1                       Backup, Database
+----
++
+// (This following block is commented out for now, it will not be included)
+// (use odo to create an odo service for the PostgreSQL Database Operator by entering the previous result in the following format: `<NAME>/<CRDs>`)
+//: <> (```shell)
+//: <> (>  odo service create postgresql-operator.v0.1.1/Database)
+//: <> (```)
+//: <> (push this service instance to the cluster)
+//: <> (```shell)
+//: <> (> odo push)
+//: <> (```)
+
+. List the service associated with the database created via the PostgreSQL Operator:
++
+[source,sh]
+----
+$ odo service list
+NAME                        AGE
+Database/sampledatabase     6m31s
+----
+
+. Create a Service Binding Request between the application and the database using the Service Binding Operator service created in the previous step `odo link` command: 
++
+[source,sh]
+----
+$ odo link Database/sampledatabase
+----
+
+. Push this link to the cluster:
++
+[source,sh]
+----
+$ odo push
+----
++
+After the link has been created and pushed a secret will have been created containing the database connection data that the application requires.
++
+You can inspect the new intermediate secret via the dashboard console in the 'my-postgresql-operator-dev4devs-com' namespace by navigating to Secrets and clicking on the secret named `mysboproj-database-sampledatabase` Notice it contains 4 pieces of data all related to the connection information for your PostgreSQL database instance.
++
+. Push the newly created link. This will terminate the existing application pod and start a new application pod.
```

Comment:
Not sure why we need this `odo push`. We already did `odo push` once after doing `odo link`. That should suffice. Another `odo push` is needed only if we're making any change to the application which, as far as I can see, we aren't.

@ajm01 any idea?

Reply:
Hi @dharmit - apologies, I didn't see these updates in my private gmail ... i think that second odo push is extraneous...so not needed at that point in the doc.

Reply:
@ajm01 OK, thanks. I'll remove that in an upcoming PR. :+1: